proto/internalapi/central/sensor_iservice.proto

```diff
@@ -45,14 +46,15 @@ message MsgFromSensor {
     ComplianceOperatorInfo compliance_operator_info = 16;
     ComplianceResponse compliance_response = 17;
     DeploymentEnhancementResponse deployment_enhancement_response = 18;
+    storage.CollectorRuntimeConfigACK collector_runtime_config_ack = 19;
```

Comment:
We would really like to get away from referencing storage objects in api protos.  Yes it is a pain to convert them all the time, but it provides us more flexibility.

Reply:
Done